auth/src/main/java/com/vimeo/networking2/internal/AuthService.kt

```diff
@@ -326,10 +406,26 @@ internal interface AuthService {
         @Header(AUTHORIZATION) authorization: String
     ): VimeoCall<Unit>
 
-    companion object {
+    private companion object {
         private const val AUTHORIZATION = "Authorization"
         private const val SCOPE = "scope"
         private const val GRANT_TYPE = "grant_type"
+        private const val REDIRECT_URI = "redirect_uri"
+        private const val STATE = "state"
+        private const val MARKETING_OPT_IN = "marketing_opt_in"
+        private const val AUTHORIZATION_CODE = "authorization_code"
+        private const val NAME = "name"
+        private const val EMAIL = "email"
+        private const val PASSWORD = "password"
+        private const val TOKEN = "token"
+        private const val TOKEN_SECRET = "token_secret"
+        private const val DOMAIN = "domain"
+        private const val USER_CODE = "user_code"
+        private const val DEVICE_CODE = "device_code"
+        private const val ACCESS_TOKEN = "access_token"
+        private const val CODE = "code"
+        private const val CLIENT_ID = "client_id"
+        private const val ID_TOKEN = "id_token"
+        private const val USERNAME = "username"
```

Comment:
I'm not sure it's worth pulling all these constants out. We originally did it with `AUTHORIZATION` since it is a constant header across all authentication methods, but maybe this could get out of hand. Not all parameters in `VimeoService` have been extracted for instance. I would like to either go all in or get rid of these constants as it looks sloppy to mix constants with inline values.

Reply:
I would rather go all in then remove them. 👍

Reply:
Converted `VimeoService` params to all constants as well.